src/interfaces/IRfqModule.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {IBaseModule} from "./IBaseModule.sol";
+import {IPerpAsset} from "v2-core/src/interfaces/IPerpAsset.sol";
+import {IAsset} from "v2-core/src/interfaces/IAsset.sol";
+
+interface IRfqModule is IBaseModule {
+  struct RfqOrder {
+    uint maxFee;
+    TradeData[] trades;
+  }
+
+  struct TakerOrder {
+    /// @dev a hash of the RfqOrder being traded against
+    bytes32 orderHash;
+    uint maxFee;
+  }
+
+  struct TradeData {
+    address asset;
+    uint subId;
+    int markPrice;
+    int amount;
+  }
+
+  struct OrderData {
```

Comment:
I think it's better to called this struct `Fill` instead of OrderData. it's kind of confusing to have another "order" struct after "maker order" and "taker order"

Reply:
i'd also just call it `price` as opposed to `markPrice` - mark price would usually imply the price you, well, mark to (e.g. mark the pnl or mark the margin), here the price is a price at which trade actually happens